junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGenerator.java

```diff
@@ -141,4 +143,97 @@ private static boolean hasParameters(Method method) {
 		}
 	}
 
+	/**
+	 * {@code DisplayNameGenerator} that generate complete sentences.
+	 *
+	 * <p>This extends the functionality of {@link ReplaceUnderscores} by
```

Comment:
This is no longer true.

Reply:
Indeed, checking all the java-docs and documentation to make sure everything is up to date